src/app/components/feed/feeds.factory.js

```diff
@@ -290,6 +292,26 @@
                 if (options.after) { options.after(); }
                 // Send the new status to remote peers
                 DataChannelService.sendStatus(that, {exclude: "picture"});
+                
```

Comment:
I would make this event more generic. Something like (just a possible example)
```
type: "channel",
data {
  channel: type,
  status: enabled,
  peerconnection: that.connection.pluginHandle.webrtcStuff.pc
}
```

The fact that video and audio are considered different (for example "paused" vs "muted") is very specific to callstats.io, not a real general difference for Jangouts. So that "if" should be in the callstats provider.